.github/workflows/build-timestamped-master.yml

```
@@ -34,15 +30,12 @@ jobs:
         env:
           packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
           packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          publishUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-          publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
-          ./gradlew publish --scan --no-daemon
+          ./gradlew clean build publish --stacktrace --scan --console=plain
       - name: Generate CodeCov Report
-        uses: codecov/codecov-action@v2
+        uses: codecov/codecov-action@v1
```

```suggestion
        uses: codecov/codecov-action@v2
```

v1 is not compatible with jacoco plugin you added